.github/workflows/CMake-minimal-version.yaml

```diff
@@ -0,0 +1,44 @@
+name: CMake minimal version test
+# The aim of this test derminate minimal supported CMake version for LibRS project
+# and check if it equlas to expected minimal CMAke version
+
+on:
+  push:
+  pull_request:
+  
+env:
+  CURRENT_CMAKE_VERSION: "Empty"
```

Comment:
See this example (using set-output):
https://stackoverflow.com/questions/57819539/github-actions-how-to-share-a-calculated-value-between-job-steps

Reply:
it's worked for me in the end